vendor/bat-native-ledger/src/bat/ledger/internal/bat_contribution.cc

```diff
@@ -1478,6 +1481,8 @@ void BatContribution::OnReconcileCompleteSuccess(
                                   ledger::ReportType::AUTO_CONTRIBUTION,
                                   probi);
     ledger_->SaveContributionInfo(probi, month, year, date, "", category);
+    contribution_count_--;
```

Comment:
can we move this at the top of the function, so that we don't need to repeat it 3 times?

Reply:
Fixed